modules/layers/src/layers/editable-geojson-layer-edit-mode-poc.js

```diff
@@ -434,23 +404,28 @@ export default class EditableGeoJsonLayer_EDIT_MODE_POC extends EditableLayer {
   }
 
   onLayerClick(event: ClickEvent) {
-    this.getActiveModeHandler().handleClick(event);
+    this.getActiveModeHandler().handleClick(event, this.getModeProps(this.props));
   }
 
   onStartDragging(event: StartDraggingEvent) {
-    this.getActiveModeHandler().handleStartDragging(event);
+    this.getActiveModeHandler().handleStartDragging(event, this.getModeProps(this.props));
   }
 
   onStopDragging(event: StopDraggingEvent) {
-    this.getActiveModeHandler().handleStopDragging(event);
+    this.getActiveModeHandler().handleStopDragging(event, this.getModeProps(this.props));
   }
 
   onPointerMove(event: PointerMoveEvent) {
-    this.getActiveModeHandler().handlePointerMove(event);
+    this.setState({ lastPointerMoveEvent: event });
+    this.getActiveModeHandler().handlePointerMove(event, this.getModeProps(this.props));
   }
 
   getCursor({ isDragging }: { isDragging: boolean }) {
```

Comment:
When will `isDragging` state update?

Reply:
`EditableLayer` is the one that keeps track of that based on pointerdown and pointerup event.